docs/reference/oev-network/searchers/submit-bids.md

```diff
@@ -0,0 +1,415 @@
+// That is to say, different versions of the auctioneer will have different bid topics.
+const getBidTopic = () => {
+  return '0x76302d70726f642d61756374696f6e6565720000000000000000000000000000';
+};
+
+// Function to encode the bid details and return to bytes
+const getBidDetails = (
+  proxyAddress,
+  condition,
+  conditionValue,
+  updaterAddress
+) => {
+  const abiCoder = new AbiCoder();
+  const BID_CONDITIONS = [
+    { onchainIndex: 0n, description: 'LTE' },
+    { onchainIndex: 1n, description: 'GTE' },
+  ];
+  const conditionIndex = BID_CONDITIONS.findIndex(
+    (c) => c.description === condition
+  );
+  return abiCoder.encode(
+    ['address', 'uint256', 'int224', 'address', 'bytes32'],
+    [
+      proxyAddress,
+      conditionIndex,
+      conditionValue,
+      updaterAddress,
+      hexlify(randomBytes(32)),
+    ]
+  );
+};
+
+const placeBid = async () => {
+  const PROXY_ADDRESS =
+    process.env.PROXY_ADDRESS ?? '0xae2debfef62b1a0c8af55dae11d197bca1bcde3f'; // Default: MNT/USD on Mantle Mainnet
+  const CHAIN_ID = process.env.CHAIN_ID ?? '5000'; // Default: mantle Mainnet
+  const BID_AMOUNT = process.env.BID_AMOUNT ?? '0.000001'; // Default: 0.000001 MNT
+  const BID_CONDITION = process.env.BID_CONDITION ?? 'LTE'; // Default: Less than or equal to
+  const BID_PRICE = process.env.BID_PRICE ?? '5'; // Default: 5 MNT
+
+  const oevNetworkProvider = new JsonRpcProvider(
+    process.env.OEV_NETWORK_RPC_URL
+  );
+  const oevNetworkWallet = Wallet.fromPhrase(process.env.MNEMONIC).connect(
+    oevNetworkProvider
+  );
+  const OevAuctionHouseArtifact =
+    await hre.artifacts.readArtifact('OevAuctionHouse');
+  const OevAuctionHouse = new Contract(
+    api3Contracts.deploymentAddresses.OevAuctionHouse['4913'],
+    OevAuctionHouseArtifact.abi,
+    oevNetworkWallet
+  );
+
+  const bidTopic = getBidTopic();
+
+  const bidDetails = getBidDetails(
+    PROXY_ADDRESS, // Proxy address of the dAPI
+    BID_CONDITION, // The condition you want to update
+    parseEther(BID_PRICE), // The price you want to update
+    deployments.OevSearcherMulticallV1, // Your deployed multicall contract address
+    hexlify(randomBytes(32)) // Random nonce
+  );
+
+  // Placing our bid on the OEV network
+  const placedbidTx = await OevAuctionHouse.placeBidWithExpiration(
+    bidTopic, // The bid topic of the auctioneer instance
+    parseInt(CHAIN_ID), // Chain ID of the dAPI proxy
+    parseEther(BID_AMOUNT), // The amount of chain native currency you are bidding to win this auction and perform the oracle update
+    bidDetails, // The details about the bid, proxy address, condition, price, your deployed multicall and random nonce
+    MaxUint256, // Collateral Basis Points is set to max
+    MaxUint256, // Protocol Fee Basis Points is set to max
+    Math.trunc(Date.now() / 1000) + 60 * 60 * 12 // Expiration time is set to 12 hours from now
```

Comment:
Nit, but it might be nice to make this a constant defined above where other inputs e.g. BID_CONDITION are defined